addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -294,6 +294,9 @@ MockServer.include({
         if (args.method === 'channel_fetch_preview') {
             return Promise.resolve(this._mockChannelFetchPreview(args));
         }
+        if (args.method === 'channel_fold') {
+            return;
+        }
```

Comment:
I would prefer an actual mock that make uses of data['mail.channel'] and changes `state`/`is_minimized`

Reply:
Out of scope? I don't need it and other mocks just return dummy promises too.

Reply:
+ we don't want to actually reimplement the whole server in js...

Reply:
It's very annoying that we cannot just rely on server routes in JS tests, and I really don't like adding magic assumptions in mockRPC... So the only option left is copying some code in mock server... Until we can finally just use python code somehow.

Changes in python related to Discuss are relatively rare, and this code is very small, so I think it's pretty acceptable to fully implement them in our tests, so that behaviour of tests are as closely consistent with reality.

(You could say tours exist for that reason, but I really don't like them because they do tests too many stuffs at the same time, and when they fail it's a really pain to debug)